addons/website_slides/models/slide_channel.py

```diff
@@ -512,7 +512,7 @@ def _action_add_members(self, target_partners, **member_values):
 
         Returns the added 'slide.channel.partner's (! as sudo !)
         """
-        to_join = self._filter_add_members(target_partners, **member_values)
+        to_join = self.sudo()._filter_add_members(target_partners, **member_values)
```

Comment:
It basically breaks the whole _filter_add_members purpose. It should be done only on SO confirmation, probably sudo the call to ``_action_add_members`` (to check if everything is ok).